src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -653,15 +653,25 @@ export default class AircraftCommander {
             return [false, readback];
         }
 
+        // TODO: move somewhere else
+        const previousAircraft = this._aircraftController.findAircraftByCallsign(runway.lastAircraftTakenoff);
```

Comment:
Adding the AircraftController into the AircraftCommander feels wrong for some reason... Perhaps it's better to give the runway access to the AircraftController so it can gain methods to return the aircraft in the runway queue, not just their IDs, and then simply use those methods here? I'm not sure, tbh... @n8rzz?

Reply:
There is more separation code to come (for example wake turbulence separation), so I definitely won't keep it here. Maybe we should create something like a `SeparationController`?

I was playing with the idea to turn `_aircraftController.findAircraftByCallsign()` into `AircraftController.findAircraftByCallsign()`.

Reply:
removed AircraftController